### ISSUE TITLE
tokenpayd - no process running

### ISSUE THREAD

**Issue by Mathiasb17** [tokenpay/tokenpay#9]
Hello,

I built tokenpay on raspbian using the instructions provided on README.md.

Calling tokenpayd seems to load some stuff on my wallet.dat and it ends with the following :

```
Loading addresses from .onion seeds
2 addresses found from .onion seeds
```

However, when i'm checking the process running on my computer :

```
user@pi:~/tokenpay $ ps aux | grep tokenpayd
pi       28704  0.0  0.0   4376   528 pts/0    S+   19:35   0:00 grep --color=auto tokenpayd
```

i can't see any process telling me that my rpi is staking anything on my address.

**Comment by marinflorin** [comment]
Hey @Mathiasb17 ,

The process has been started by "user"? Can you also try a `sudo ps aux | grep tokenpayd` ?

**Comment by justinvforvendetta** [comment]
yeah, sudo would let you know if it was started.. also, what command are you starting it with?
try: ./tokenpayd -daemon (and make sure your  .conf is properly created)

**Comment by gabriel-samfira** [comment]
I created a small ```readme``` on how to set up a ```systemd``` service for the wallet, and proposed it in a PR. Until it merges, you can access it at:

https://github.com/gabriel-samfira/tokenpay/blob/add-contrib/doc/systemd-service-setup.md

This should ensure your wallet starts up automatically when your raspberry pi/server boots.

**Comment by Mathiasb17** [comment]
Hey guys, thanks a lot for your reactivity.

I started the daemon with `tokenpayd` (i added it in my `$PATH`).

@marinflorin I also tried `sudo ps aux | grep tokenpayd` but i have the same result as without `sudo`.

@justinvforvendetta I also tried `tokenpayd -daemon` but i have the same result as with `tokenpayd`.

@gabriel-samfira Thanks for your reply, i'm gonna try this right now, will edit this post asap.
